Signal/src/call/PeerConnectionClient.swift

```diff
@@ -115,7 +115,14 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
 
     private var videoSender: RTCRtpSender?
     private var localVideoTrack: RTCVideoTrack?
-    private weak var remoteVideoTrack: RTCVideoTrack?
+    // RTCVideoTrack is fragile and prone to throwing exceptions and/or
```

Comment:
Ah, I see why.  All of those other properties are only ever mutated on the main thread during the PCC constructor.  By definition PCC.terminate() can't be called until after that constructor returns. So I think we're safe here.